src/ck3/tables/effects.rs

```diff
@@ -507,7 +507,7 @@ const SCOPE_EFFECT: &[(Scopes, &str, Effect)] = &[
     (Scopes::Character, "refund_perks", Item(Item::Lifestyle)),
     (Scopes::Character, "release_from_prison", Yes),
     (Scopes::Accolade, "remove_acclaimed_knight", Yes),
-    (Scopes::Scheme, "remove_agent_slot", Item(Item::AgentType)),
+    (Scopes::Scheme, "remove_agent_slot", Scope(Item::AgentType)),
```

Comment:
It should now be `Scopes::AgentSlot` instead of the item type :)

Reply:
(Scopes::Scheme, "remove_agent_slot", Scope(Scopes::AgentType)),